# Type: feat
# Subtype: Ex
# English Summary: Added getPadX/Y to Table
# Chinese Summary: Table，添加 getPadX/Y
# Entry Number: 25

# === Git Diff Content ===

diff --git a/gdx/src/com/badlogic/gdx/scenes/scene2d/ui/Table.java b/gdx/src/com/badlogic/gdx/scenes/scene2d/ui/Table.java
index e2f6bf3..5dfda92 100644
--- a/gdx/src/com/badlogic/gdx/scenes/scene2d/ui/Table.java
+++ b/gdx/src/com/badlogic/gdx/scenes/scene2d/ui/Table.java
@@ -27,7 +27,10 @@
 
 package com.badlogic.gdx.scenes.scene2d.ui;
 
-import java.util.List;
+import com.esotericsoftware.tablelayout.BaseTableLayout.Debug;
+import com.esotericsoftware.tablelayout.Cell;
+import com.esotericsoftware.tablelayout.Toolkit;
+import com.esotericsoftware.tablelayout.Value;
 
 import com.badlogic.gdx.graphics.Color;
 import com.badlogic.gdx.graphics.g2d.SpriteBatch;
@@ -42,10 +45,8 @@ import com.badlogic.gdx.scenes.scene2d.utils.Align;
 import com.badlogic.gdx.scenes.scene2d.utils.Drawable;
 import com.badlogic.gdx.scenes.scene2d.utils.ScissorStack;
 import com.badlogic.gdx.utils.Array;
-import com.esotericsoftware.tablelayout.BaseTableLayout.Debug;
-import com.esotericsoftware.tablelayout.Cell;
-import com.esotericsoftware.tablelayout.Toolkit;
-import com.esotericsoftware.tablelayout.Value;
+
+import java.util.List;
 
 /** A group that sizes and positions children using table constraints. By default, {@link #getTouchable()} is
  * {@link Touchable#childrenOnly}.
@@ -434,6 +435,16 @@ public class Table extends WidgetGroup {
 		return layout.getPadRight();
 	}
 
+	/** Returns {@link #getPadLeft()} plus {@link #getPadRight()}. */
+	public float getPadX () {
+		return layout.getPadLeft() + layout.getPadRight();
+	}
+
+	/** Returns {@link #getPadTop()} plus {@link #getPadBottom()}. */
+	public float getPadY () {
+		return layout.getPadTop() + layout.getPadBottom();
+	}
+
 	public int getAlign () {
 		return layout.getAlign();
 	}
